code/game/machinery/doors/airlock.dm

```diff
@@ -937,21 +937,32 @@ About the new airlock wires panel:
 					user.visible_message(SPAN_WARNING("[user] headbutts the airlock. Good thing they're wearing a helmet."))
 				return
 
-		if(H.species.can_shred(H))
+		if(H.a_intent == I_HURT)
+			var/shredding = H.species.can_shred(H)
+			var/can_crowbar = H.default_attack?.crowbar_door && (stat & (BROKEN|NOPOWER))
+			if(shredding || can_crowbar)
+				if(!density)
+					return
 
-			if(!src.density)
-				return
+				H.visible_message("<b>[H]</b> begins to pry open \the [src]!", SPAN_NOTICE("You begin to pry open \the [src]!"), SPAN_WARNING("You hear the sound of an airlock being forced open."))
 
-			H.visible_message("<b>[H]</b> begins to pry open \the [src]!", SPAN_NOTICE("You begin to pry open \the [src]!"), SPAN_WARNING("You hear the sound of an airlock being forced open."))
+				if(!do_after(H, 120, 1, act_target = src))
+					return
 
-			if(!do_after(H, 120, 1, act_target = src))
-				return
+				var/check = src.open(1)
 
-			src.do_animate("spark")
-			src.stat |= BROKEN
-			var/check = src.open(1)
-			H.visible_message("<b>[H]</b> slices \the [src]'s controls, [check ? "ripping it open" : "breaking it"]!", SPAN_NOTICE("You slice \the [src]'s controls, [check ? "ripping it open" : "breaking it"]!"), SPAN_WARNING("You hear something sparking."))
-			return
+				if(shredding)
+					src.do_animate("spark")
+					src.stat |= BROKEN
+					H.visible_message("<b>[H]</b> slices \the [src]'s controls, [check ? "ripping it open" : "breaking it"]!", SPAN_NOTICE("You slice \the [src]'s controls, [check ? "ripping it open" : "breaking it"]!"), SPAN_WARNING("You hear something sparking."))
+				return
+			if(H.default_attack?.attack_door && !(stat & (BROKEN|NOPOWER)))
+				user.visible_message(SPAN_DANGER("\The [user] forcefully strikes \the [src] with their [H.default_attack.attack_name]!"))
+				user.do_attack_animation(src, null)
+				playsound(loc, hitsound, 60, 1)
+				take_damage(H.default_attack.damage * 3)
```

Comment:
This seems like a questionable amount of damage.

Reply:
It's better to not hijack values like that if they're too low - it can cause unexpected consequences. I'd say that this should be a different value altogether, like `breaking_proficiency` or whatever.